src/openads-appnexus/PrebidClientImpl.js

```diff
@@ -42,4 +42,14 @@ export default class PrebidClientImpl {
     this._pbjs.que.push(() => this._pbjs.setConfig(config))
     return this
   }
+
+  setBidderSettings(bidderSettings) {
```

Comment:
I was not sure why there was a change of 'config' separating core and bidderSettings when we talked about this, reviewing the code I'm still not sure why there's a logical separation when we're setting the client config and when we're setting the bidderSettings..

Why are we setting this on the bidders request? (each request)
As I see it, the bidderSettings are configuration like the core settings, and it won't change between display/refresh calls, so: **should'nt it be set when the pbjs is configured together with the core config?**

Reply:
Official documentation does not explain this in a clear way, but **bidderSettings** object must be overridden everytime after a call to **pbjs.requestBids** method is done. If you remember, same thing happened when we were talking about the point to call **pbjs.setConfig** method, it must be done in a precise order. As I said, it is not clear enough in the official documentation.

Apart from that pbjs does not allow setting both "configurations" at the same time using the same method **pbjs.setConfig** as bidderSettings are not included in core configuration (as they called it). In fact, pbjs does not offer any method to update bidderSettings, forcing us to it as follows:

```javascript
pbjs.bidderSettings = bidderSettings

// being bidderSettings an object with a defined structure
```

We decided to include both core configuration and bidderSettings as a whole connector configuracion under **prebidConfig** node as we identified both as configuration parameters.

Reply:
@alextremp please have a look after the agreed refactor.